internal/tests/passingvalues_test.go

```diff
@@ -0,0 +1,138 @@
+package tests
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/conductorone/baton-sdk/pkg/field"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPassingValuesToFieldsViaCLI(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Millisecond*10)
```

Comment:
I see `time.Millisecond*10` in quite a few places. It should probably be a const so any updates to the timeout don't require a big diff.

Also why 10 milliseconds and not 1 or 100? I tried different values and anything above  20ms seems to spit out errors like `panic: unexpected call to os.Exit(0) during test`.

Reply:
> Also why 10 milliseconds and not 1 or 100?

it was on a whim I chose that number lol